script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt

```diff
@@ -42,20 +42,20 @@ I didn't expect the Tohno family to call me back at all. Keiko-san, the rest of
 "You didn't think they would call you back?
 Even though you're the eldest son of the Tohno family?"
 <Page13>
-"Yes... to tell you the truth, my recovery was just a front. My father and I never got along. It's like he threw me out of the house because of my injury."
+"Yes... to tell you the truth, my recovery was just a front. My father and I never got along. It's like he threw me out of the house pretexting my injury to keep up appearances."//「はい。……本当の事を言ってしまうと、療養生活というのは建前なんです。俺と親父は犬猿の仲というヤツでして。怪我を理由に<体|てい>よく放り出されたようなものなんです」
```

Comment:
Not sure `pretexting` is a word
> It's more like he threw me out of the house, using my injury as a pretext to keep up appearances.

Reply:
Whoops haha, I got influenced by French it seems!